ExampleApps/UIKitExample/SwiftCurrent_UIKitTests/TestUtilities/NotificationReceiverLocal.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  NotificationReceiverLocal.swift
```

Comment:
This thing and things related to it (data, protocol, this receiver) should all move into `SwiftCurrent_Test` and be a part of the testing suite of tools.  I haven't done it yet, because I'm looking for feedback on more tweaks or value of this.  I think it's not complete but if we kept polishing, this would be a decent way of providing flexibility to people testing, while abstracting away the nitty-gritty of doing it.

There's probably too much overlap with the Local version and the Test version of this receive.